Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,72 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_thinking_of_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_thinking_of_you:
+    if not renpy.seen_label("mas_compliment_thinking_of_you_2"):
+        call mas_compliment_thinking_of_you_2
+    else:
+        call mas_compliment_thinking_of_you_3
+    return
+
+label mas_compliment_thinking_of_you_2:
+    m 1wubso "..."
+    m 1wubfb "You..."
+        
+    if mas_isMoniNormal:
+        m 2hubfb "You have no idea what that means to me, [player]."
+        
+    elif mas_isMoniUpset:
+        m "That you would say that..."
+        m "It means the world to me, [player]."
+        
+    elif mas_isMoniDis:    
+        m 1ekbfa "..."
+        m "That's such a relief."
+        m "Thank you."
+        
+    elif mas_isMoniBroken:
+        m 1rksdld "Hmmm... Thank you."
+        m 1rksdla "..."
+        $ mas_gainAffection(2,bypass=True)
+        
+        return
+        
+    menu:
+        "You're very presence fills the corners of my imagination":
```

Comment:
**Your**

Too much, I think. 

As for suggestions, in the most neutral way possible, how about spending more than exactly 1 hour on topics.

Reply:
Eh, fair. I feel I'm still trying to get a good grip on who Monika is, to be fully honest.